examples/js/loaders/FBXLoader.js

```diff
@@ -385,6 +385,21 @@
 
 		texture.wrapS = valueU === 0 ? THREE.RepeatWrapping : THREE.ClampToEdgeWrapping;
 		texture.wrapT = valueV === 0 ? THREE.RepeatWrapping : THREE.ClampToEdgeWrapping;
+        
+        if ( textureNode.properties.Scaling ) {
+
+            var values = textureNode.properties.Scaling.value
+
+            if ( typeof values === "string" ) {
+                values = values.split(",").map(function(number) {
```

Comment:
Thanks for your contribution! I just wanna give you some code style feedback 😉 : Please have look at [Mr.doob's Code Style™](https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style%E2%84%A2) for correct code style. In your case:
```javascript
if ( textureNode.properties.Scaling ) {
   
   values = textureNode.properties.Scaling.value

   if ( typeof values === 'string' ) {

      values = values.split( ',' ).map( function( number ) {

         return parseFloat( number );

      } );
   
   }
            
   texture.repeat.x = values[ 0 ];
   texture.repeat.y = values[ 1 ];
            
}
```
Also don't forget to use semicolons and hard instead of soft tabs. You can also use an ESLint plugin with `mdcs` preset or the [online beautifier](http://zz85.github.io/mrdoobapproves/) to check your code.